src/main/openapi/openapi.yaml

```diff
@@ -109,7 +109,7 @@ paths:
                       |JSONL|Newline separated single-line JSON representations of Documents.|
```

Comment:
Please add a description for TXT. Maybe mention it will be in UTF-8?

Reply:
Description added.